content/en/tracing/trace_collection/compatibility/dotnet-framework.md

```diff
@@ -95,6 +95,22 @@ The [latest version of the .NET Tracer][5] can automatically instrument the foll
 
 Don't see the library you're looking for? First, check if the library produces observability data compatible with OpenTelemetry (see [Using OpenTelemetry Instrumentation Libraries][11] for more details). If not, Datadog is continually adding additional support. [Check with the Datadog team][6] for help.
 
+## OpenTelemetry based integrations
```

Comment:
I didn't expect to see new content here - is this part of a bad merge?

Reply:
This must be a bad merge. This content was removed in a recent PR so it looks like added content. @brett0000FF please remove this section entirely

Reply:
Good catch--thanks! Removed these sections.